viewlet/tests/urls.py

```diff
@@ -1,11 +1,20 @@
 # coding=utf-8
 from __future__ import unicode_literals
-try:
-    from django.conf.urls import patterns, include
-except ImportError:
-    from django.conf.urls.defaults import patterns, include
+import django
 
-urlpatterns = patterns(
-    '',
-    (r'^viewlet/', include('viewlet.urls')),
-)
+if django.VERSION[:2] < (1, 7):
+    try:
+        from django.conf.urls import patterns, include
+    except ImportError:
+        from django.conf.urls.defaults import patterns, include
```

Comment:
We can use `compat.urlpatterns` and I think we can do the similar thing for the rest if we move them there. It will help to clean code in many places